dotnet/src/SemanticKernel.Skills/Skills.Grpc/Skills.Grpc.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+  </PropertyGroup>
+  <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.SemanticKernel.Skills.Grpc</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Skills.Grpc</RootNamespace>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
We recently downgraded target framework for all SK SDK projects to netstandard2.0 to support wider range of clients. Please downgrade it to the netstandard2.0. CC: @shawncal

Reply:
This is not compatible with the latest Grpc.Net.Client. Looks like [quite a few projects have to be downgraded](https://github.com/search?q=repo%3Amicrosoft%2Fsemantic-kernel%20%3CTargetFramework%3Enetstandard2.1%3C%2FTargetFramework%3E&type=code)

![image](https://user-images.githubusercontent.com/18152044/232108619-89d1be33-3381-4ad5-a2dc-90d2edcc9bef.png)

Reply:
FYI - IntelliJ Rider was showing the build as successful, however, on VS, it wasn't the case (likely because Rider had some nuget cache; going forward, I'll validate on VS and cli) - resolved this over a call with @SergeyMenshykh. 

`netstandard2.1` is also okay for now, a per discussion with @shawncal. Marking this as resolved.